app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1><%= "#{@recipe.name}" %></h1>
+<h3>Ingredients</h3>
+<% @recipe.ingredients.each do |ingredient| %>
+  <li><%= ingredient.name %></li>
+<% end %>
+<h3>Total Cost: <%= @recipe.total_cost %></h3>
+<h3>Complexity</h3>
+<%= "#{@recipe.complexity}/10" %>
+<h3>Genre</h3>
+<%= @recipe.genre %>
+<%= form_with url: "/recipe_ingredients", method: :post, data: {turbo: false}, local: true do |f| %>
```

Comment:
`/recipe_ingredients` is a perfectly RESTful path, but another RESTful option is: `/recipes/:recipe_id/ingredients` which would allow you to pass the recipe ID in the path rather than having to pass it as a hidden field. Both options are RESTful!